### DISCUSSION TITLE
Lighthouse deck broken?

### DISCUSSION THREAD

**Discussion by matejkarasek** [bitcraze/discussions#1692] [Q&A]
Hi, one of our Lighthouse deck started acting up. I replaced the two 10pin female connectors, but it still behaves strangely.

https://github.com/user-attachments/assets/a502963c-730b-4a7d-8f23-4ee4f902a372

This starts when I touch it/press it and can be reproduced on different crazyflies/bolts.

Any ideas what might be wrong and if it can be repaired?

Thanks,
Matej

**Comment by tobbeanton** [comment]
My guess is that there is a HW issue if it is one off. You can not replicate it with another LH-deck? The STM32 is [controlling the LED:s ](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/modules/src/lighthouse/lighthouse_core.c#L160)and it seems to be jumping between not receiving and passing it to the kalman filter.

**Comment by matejkarasek** [comment]
Thanks, yes, the problem is only with this one particular deck.

I have replaced the female 10-pin connectors but it did not help. But it seems to be some bad contact somewhere, as the issue gets triggered e.g. when I touch it. I guess I just retire this one then :)

**Reply by tobbeanton** [reply]
It might be interesting fo us to check it out. Perhaps you can send it to us with the next convinient shipment?

**Reply by matejkarasek** [reply]
Will do!
